roles/container-engine/docker/templates/docker.service.j2

```diff
@@ -30,9 +30,7 @@ ExecStart={{ docker_bin_dir }}/docker{% if installed_docker_version.stdout is ve
 {% endif %}
           $DOCKER_OPTS \
           $DOCKER_STORAGE_OPTIONS \
-          $DOCKER_NETWORK_OPTIONS \
           $DOCKER_DNS_OPTIONS \
```

Comment:
CI is breaking, I guess you need to remove the `\`

Reply:
Oops, will fix it in a min